plugins/optimization-detective/readme.txt

```diff
@@ -205,7 +205,7 @@ The [plugin source code](https://github.com/WordPress/performance/tree/trunk/plu
 
 **Enhancements**
 
-* Log URL metrics group collection to console when debugging is enabled (`WP_DEBUG` is true). ([1295](https://github.com/WordPress/performance/pull/1295))
+* Log URL metric group collection to console when debugging is enabled (`WP_DEBUG` is true). ([1295](https://github.com/WordPress/performance/pull/1295))
```

Comment:
Nit-pick, but I don't think we should update older changelogs as they are "history artifacts" (e.g. we probably wouldn't update this in an old GitHub release either). Let's revert this.

Reply:
Reverted in ef9f3ae7